packages/cerebral-website-tutorial/parts/01_get_started/src/components/HeaderButton/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
```

Comment:
So I can just start with `src/` and copy files form `parts/01-get-started/` to get next step?